internal/cron/cron_test.go

```diff
@@ -0,0 +1,221 @@
+package cron
+
+import (
+	"testing"
+	"time"
+
+	"github.com/alecthomas/assert/v2"
+)
+
+func TestNonUTC(t *testing.T) {
+	// This cron package only works with UTC times.
+	// Passing in non-UTC times works fine, but the results will be in UTC.
+}
+
+func TestParsingAndValidationErrors(t *testing.T) {
+	// Rather than testing successful parsing, test them in TestNext()
+	for _, tt := range []struct {
+		str string
+		err string
+	}{
+		{"* * * *", "1:8: unexpected token \"<EOF>\" (expected Component Component? Component?)"},
```

Comment:
To get better errors, it might be an idea to modify the parser to accept an arbitrary number of elements, then validate the correct number are provided  and return custom errors.